app/code/Magebit/Faq/Controller/Adminhtml/Question/Delete.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Magebit_Faq
+ *
+ * @category     Magebit
+ * @package      Magebit_Faq
+ * @author       Artjoms Fomins <info@magebit.com>
+ * @copyright    Copyright (c) 2023 Magebit, Ltd.(https://www.magebit.com/)
+ */
+
+declare(strict_types=1);
+
+namespace Magebit\Faq\Controller\Adminhtml\Question;
+
+use Magento\Framework\App\Action\HttpPostActionInterface;
+
+/**
+ * Class that implement delete logic
+ */
+class Delete extends \Magento\Backend\App\Action implements HttpPostActionInterface
+{
+    /**
+     * @param \Magento\Backend\App\Action\Context $context
+     * @param \Magento\Framework\Registry $coreRegistry
+     */
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Framework\Registry $coreRegistry
```

Comment:
**Requirements**

- The newly created code follows the best practices of Magento development and doesn't contain any junk files or code.
- All files and methods are documented with phpDoc comments.
- The end result is in line with the description and images provided above.